### PR TITLE
Add adjusted error message for statistics page

### PR DESCRIPTION
**Pull request by SteffiStuffel**
### Short description

Adjusted the error message. 

### Resolved issues

Fixes: #1700 

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Review by timoludwig — COMMENTED** [review]
Thanks a lot! :+1:

This is definitely an improvement, I was just wondering whether this is now misleading in the case that something else went wrong and it's not the Matomo timeout causing the error...
Do you think it is worth looking into the different possible exceptions the [MatomoApiManager](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/cms/utils/matomo_api_manager.py) can throw and then differentiating them [here](https://github.com/digitalfabrik/integreat-cms/blob/15ce171cc8041bac2d24823e271632db56eaebf1/integreat_cms/cms/views/statistics/statistics_actions.py#L94-L96) to show two different kind of server error messages in the frontend [here](https://github.com/digitalfabrik/integreat-cms/blob/15ce171cc8041bac2d24823e271632db56eaebf1/integreat_cms/cms/templates/statistics/statistics_overview.html#L19)?
Not sure whether it's worth the effort though.

**Comment by timoludwig** [comment]
@SteffiStuffel sorry for misleading you, I just noticed that the MatomoApiManager does in fact not handle timeout errors gracefully since the exception thrown is an `asyncio.TimeoutError` which is no subclass of `aiohttp.ClientError`.
This means we could e.g. handle the error directly in the view with:
```
    except asyncio.TimeoutError:
        return JsonResponse(
            {"error": "Timeout during request to Matomo API"}, status=504
        )
```
in the views [here](https://github.com/digitalfabrik/integreat-cms/blob/15ce171cc8041bac2d24823e271632db56eaebf1/integreat_cms/cms/views/statistics/statistics_actions.py#L47-L51) and [here](https://github.com/digitalfabrik/integreat-cms/blob/15ce171cc8041bac2d24823e271632db56eaebf1/integreat_cms/cms/views/statistics/statistics_actions.py#L92-L96) (also add `import asyncio`) and then show the corresponding div based on the return status code.

You can test the code by replacing
```
        async with aiohttp.ClientSession() as session:
```
by
```
        timeout = aiohttp.ClientTimeout(total=1)  # 1 sec timeout
        async with aiohttp.ClientSession(timeout=timeout) as session:
```
in the API manager.

**Comment by SteffiStuffel** [comment]
@timoludwig thanks for explanation, adjusted it like you suggested.

File: integreat_cms/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -6186,42 +6188,47 @@ msgstr "Neuen Schlüssel hinzufügen"
 msgid "Number of online accesses over the last 14 days."
 msgstr "Anzahl der Online-Zugriffe der letzten 14 Tage."
 
+#: cms/templates/statistics/_statistics_widget.html:23
+#: cms/templates/statistics/statistics_overview.html:19
+msgid "The statistics network is currently experiencing heavy traffic."
+msgstr "Der Statsitik-Server hat aktuell eine hohe Auslastung."
```
Comment (timoludwig) [inline integreat_cms/locale/de/LC_MESSAGES/django.po:6194]:
```suggestion
msgstr "Der Statistik-Server hat aktuell eine hohe Auslastung."
```
Small typo

**Review by timoludwig — CHANGES_REQUESTED** [review]
Thanks a lot! :+1:
Just found a small typo.

**Review by timoludwig — APPROVED** [review]
Thanks! :ok_hand:

**Pull request merged**